### PR TITLE
Handle remove brick correctly when subvolume number shifts

### PR DESCRIPTION
**Pull request by nnDarshan**
As part of remove brick handler we are deleting all the brick
related information, so that next sync takes care of updating
the correct info

tendrl-bug-id: Tendrl/gluster-integration#494
Signed-off-by: Darshan N <darshan.n.2024@gmail.com>

### CODE REVIEW

File: tendrl/gluster_integration/message/callback.py [hunk 1]
```diff
@@ -19,6 +19,7 @@
 RESOURCE_TYPE_BRICK = "brick"
 RESOURCE_TYPE_PEER = "host"
 RESOURCE_TYPE_VOLUME = "volume"
+SYNC_INTERVAL = NS.config.data.get("sync_interval", 10)
```
Comment (GowthamShanmugam) [inline tendrl/gluster_integration/message/callback.py]: It will give error, NS is not available at this stage

Comment (GowthamShanmugam) [reply tendrl/gluster_integration/message/callback.py]: callback import is happening before NS create https://github.com/Tendrl/gluster-integration/blob/master/tendrl/gluster_integration/manager/__init__.py#L13.


So we can't use NS  here

Comment (nnDarshan) [reply tendrl/gluster_integration/message/callback.py]: Thanks for pointing that.

**Comment by GowthamShanmugam** [comment]
LGTM

**Comment by nnDarshan** [comment]
@r0h4n your changes look good

**Pull request merged**